.github/actionlint.yaml

```diff
@@ -0,0 +1,4 @@
+self-hosted-runner:
+  # Labels of self-hosted runner in array of strings.
+  labels:
+    - m1mac
```

Comment:
?

Reply:
But I guess this file can be removed since we don't check workflows with actionlint in this repository.

Reply:
hmm this should be run on an ubuntu-latest don't you think ?

Reply:
wait that's not how you setup the actionlint stuff before was it ?

Reply:
ok nevermind don't know what I read here, sorry brain fart